source/community/reactnative/android/app/src/main/java/com/rnadaptivecards/MainActivity.kt

```diff
@@ -0,0 +1,22 @@
+package com.rnadaptivecards
+ 
+import com.facebook.react.ReactActivity
+import com.facebook.react.ReactActivityDelegate
+import com.facebook.react.defaults.DefaultNewArchitectureEntryPoint.fabricEnabled
+import com.facebook.react.defaults.DefaultReactActivityDelegate
+ 
+class MainActivity : ReactActivity() {
+ 
+  /**
+   * Returns the name of the main component registered from JavaScript. This is used to schedule
+   * rendering of the component.
+   */
+  override fun getMainComponentName(): String = "RNAdaptiveCards"
+ 
+  /**
+   * Returns the instance of the [ReactActivityDelegate]. We use [DefaultReactActivityDelegate]
+   * which allows you to enable New Architecture with a single boolean flags [fabricEnabled]
```

Comment:
which allows to enable New Architecture with a single boolean flags [fabricEnabled]

(remove you)